src/openklant/management/commands/migrate_to_v2_phonenumbers.py

```diff
@@ -0,0 +1,244 @@
+    "vestiging": "vestigings_nummer",
+}
+
+
+def _save_klanten_phonenumbers(
+    url: str, token: TokenAuth, klanten: list[Klant]
+) -> list[str]:
+    if not klanten:
+        logger.info("no_klanten_to_save")
+        return []
+
+    logger.info("updating_klanten_v2_api", count=len(klanten))
+
+    openklant_client = OpenKlantClient(url, token)
+    updated_klanten = []
+
+    for klant in klanten:
+        subject_type = getattr(klant, "subject_type", None)
+        nummer_field = NUMMER_FIELD_MAPPING.get(subject_type)
+        nummer = getattr(klant.subject_identificatie, "get", lambda x: None)(
+            nummer_field
+        )
+        try:
+            soort_partij = klant._get_soort_partij()
+            partij = Partij.objects.get(soort_partij=soort_partij, nummer=nummer)
+        except Partij.DoesNotExist:
+            logger.warning(
+                "partij_not_found",
+                klant=klant,
+                subject_type=subject_type,
+                nummer=nummer,
+            )
+            continue
+        existing_da = DigitaalAdres.objects.filter(
+            partij=partij,
+            soort_digitaal_adres="telefoonnummer",
+            referentie="portaalvoorkeur",
+        ).first()
+
+        if existing_da:
+            logger.info(
+                "digitaaladres_exists",
+                partij=partij.pk,
+                nummer=nummer,
+            )
+            continue
+
+        digitaal_adres = klant.to_digitaal_adres_phonenumber()
+        if digitaal_adres:
+            digitaal_adres_data = digitaal_adres.dict()
+            digitaal_adres_data["referentie"] = "portaalvoorkeur"
+
+            _data = openklant_client.create(
+                DIGITALE_ADDRESSEN_PATH,
+                digitaal_adres_data,
+            )
+
+            if _data and "url" in _data:
+                try:
+                    da = DigitaalAdres.objects.get(uuid=_data["uuid"])
+                    da.partij = partij
+                    da.save(update_fields=["partij"])
+                    updated_klanten.append(_data["url"])
+                except DigitaalAdres.DoesNotExist:
+                    logger.error(
+                        "digitaaladres_not_found_after_create", uuid=_data["uuid"]
+                    )
+
+    return updated_klanten
+
+
+def _generate_dummy_token() -> str:
+    try:
+        token_auth, _ = TokenAuth.objects.get_or_create(
+            application=MIGRATION_TOKEN_IDENTIFIER
+        )
+    except TokenAuth.MultipleObjectsReturned:
+        dummy_tokens = TokenAuth.objects.get_or_create(
+            application=MIGRATION_TOKEN_IDENTIFIER
+        )
+
+        token_auth = dummy_tokens.first()
+        deletion_tokens = dummy_tokens.exclude(pk=token_auth.pk)
+        deleted_token_ids = list(deletion_tokens.values_list("id", flat=True))
+        deletion_tokens.delete()
+
+        logger.warning("removed_migration_dummy_tokens", tokens=deleted_token_ids)
+    return token_auth.token
```

Comment:
Is this the same as `_generate_dummy_token` in `migrate_to_v2`? If so, could you move them to the `utils.py` file in `openklant/migration`?